__tests__/api.test.js

```diff
@@ -230,6 +246,52 @@ describe('POST /api/articles/:article_id/comments', () => {
   });
 });
   
+describe('PATCH /API/articles/:article_id', () => {
+  test('200: responds with the updated article', () => {
+    return request(app)
+    .patch('/api/articles/1')
+    .send({inc_votes: 9})
+    .expect(200)
+    .then(({body}) => {
+      expect(body.article).toMatchObject({
+        article_id: 1,
+        title: expect.any(String),
+        topic: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String),
+        created_at: expect.any(String),
+        votes: 109,
+      })
+    })
+  });
+  test('400: responds with error msg: inv_votes is not a number', () => {
+    return request(app)
+    .patch('/api/articles/1')
+    .send({inc_votes: 'banana'})
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe('Votes must be a number')
+    })
+  });
+  test('404: responds with error msg when article not exists', () => {
+    return request(app)
+    .patch('/api/articles/19999')
+    .send({inc_votes: 1})
+  .expect(404)
+  .then(({body}) => {
+    expect(body.msg).toBe('Article not found')
+  })
+  })
+  test('400: responds with error msg when article not exists', () => {
+    return request(app)
+    .patch('/api/articles/invalid')
+    .send({inc_votes: 1})
+  .expect(400)
+  .then(({body}) => {
+    expect(body.msg).toBe('Invalid ID')
+  })
+  })
+});
```

Comment:
Great tests Jian. One more test would be helpful though. What if the inc_votes object is empty?